README.md

```diff
@@ -1,67 +1,76 @@
+### Installing Key Vault FlexVolume
 
-💡 Make sure you have a Kubernetes cluster
+#### OPTION 1: New AKS Engine cluster
 
-### Install the KeyVault Flexvolume ###
+[AKS Engine] creates customized Kubernetes clusters on Azure.
 
-#### OPTION 1 - AKS-Engine addon ####
+Follow the AKS Engine [add-on documentation] to create a new Kubernetes cluster with Key Vault FlexVolume already deployed.
 
-Follow [this](https://github.com/Azure/aks-engine/blob/master/examples/addons/keyvault-flexvolume/README.md) to use aks-engine to create a new Kubernetes cluster with the Key Vault FlexVolume already deployed.
+#### OPTION 2: Existing AKS cluster
 
-#### OPTION 2 - AKS (Azure Kubernetes Service) Manually ####
+Azure Kubernetes Service ([AKS]) creates managed, supported Kubernetes clusters on Azure.
 
-```bash
+Deploy Key Vault FlexVolume to your AKS cluster with this command:
 
+```bash
 kubectl create -f https://raw.githubusercontent.com/Azure/kubernetes-keyvault-flexvol/master/deployment/kv-flexvol-installer.yaml
 ```
-To validate the installer is running as expected, run the following commands:
 
-```bash
-kubectl get pods -n kv
-```
+ To validate Key Vault FlexVolume is running as expected, run the following command:
 
-You should see the keyvault flexvolume installer pods running on each agent node:
+ ```bash
+ kubectl get pods -n kv
+ ```
 
-```bash
-keyvault-flexvolume-f7bx8   1/1       Running   0          3m
-keyvault-flexvolume-rcxbl   1/1       Running   0          3m
-keyvault-flexvolume-z6jm6   1/1       Running   0          3m
-```
-### Use the KeyVault FlexVolume ###
+ The output should show `keyvault-flexvolume` pods running on each agent node:
 
-The KeyVault FlexVolume offers two modes for accessing a Key Vault instance: Service Principal and Pod Identity.
+  ```bash
+  NAME                        READY     STATUS    RESTARTS   AGE
+  keyvault-flexvolume-f7bx8   1/1       Running   0          3m
+  keyvault-flexvolume-rcxbl   1/1       Running   0          3m
+  keyvault-flexvolume-z6jm6   1/1       Running   0          3m
+  ```
 
-#### OPTION 1 - Service Principal ####
+### Using Key Vault FlexVolume
 
-Add your service principal credentials as a Kubernetes secrets accessible by the KeyVault FlexVolume driver.
+Key Vault FlexVolume offers two modes for accessing a Key Vault instance: [Service Principal] and [Pod Identity].
+
+#### OPTION 1: Service Principal
+
+Add your service principal credentials as Kubernetes secrets accessible by the Key Vault FlexVolume driver.
 
 ```bash
 kubectl create secret generic kvcreds --from-literal clientid=<CLIENTID> --from-literal clientsecret=<CLIENTSECRET> --type=azure/kv
 ```
 
-Ensure this service principal has all the required permissions to access content in your key vault instance.
-If not, you can run the following using the Azure cli:
+Ensure this service principal has all the required permissions to access content in your Key Vault instance.
+If not, run the following [Azure CLI] commands:
```

Comment:
The `Assign Azure Identity Roles` section below should be part of this section as well since we need to ` Ensure that your Azure Identity has the role assignments required to see your Key Vault instance and to access its content`